packages/clerk-js/setupJestAfterEnv.ts

```diff
@@ -1,11 +1 @@
 import '@testing-library/jest-dom';
```

Comment:
We had `@testing-library/jest-dom/extend-expect` in our testUtils file. This line already covers this